src/pyscipopt/scip.pxi

```diff
@@ -2834,6 +2834,35 @@ cdef class Model:
         PY_SCIP_CALL(SCIPreleaseCons(self._scip, &scip_cons))
 
         return pyCons
+    
+    def addPiecewiseLinearCons(self, X, Y, a, b):
+        """add piecewise relation with multiple selection formulation
+
+            :param a: array with x-coordinates of the points in the piecewise linear relation
```

Comment:
why not name these X and Y?

Reply:
Just realized I forgot to add the description of X and Y in the comment.

But a and b are parameter arrays, X and Y are the model variables themselves.